### ISSUE TITLE
problem check laravel pach in windows

### ISSUE THREAD

**Issue by fazelit** [naifalshaye/laravelgui#3]
Laravel's directory path is not verified
in windows 10
image: 
`http://uupload.ir/files/q8sy_annotation_(2).png`
![Annotation (2)](https://user-images.githubusercontent.com/5165331/59140531-1ae75e80-89b4-11e9-98f8-e899acc4c637.png)

**Comment by naifalshaye** [comment]
Make sure PHP path is added to your environment variables.

**Comment by fazelit** [comment]
> Make sure PHP path is added to your environment variables.

yes, php path is added to my environment var. But it does not work.
![image](https://user-images.githubusercontent.com/5165331/59152223-20e14c00-8a55-11e9-85a7-7638b6e62671.png)

**Comment by naifalshaye** [comment]
It's been tested on Windows 10 successfully.
Double check the path select the Laravel project root directory
In terminal go to Laravel path and make sure you get an output when you run "php artisan commands"

**Comment by fazelit** [comment]
> In terminal go to Laravel path and make sure you get an output when you run "php artisan commands"

test php artisan list, but did not work.
![image](https://user-images.githubusercontent.com/5165331/59164150-92d19800-8b1e-11e9-8e54-96269a009f64.png)
